misaligned.js

```diff
@@ -1,16 +1,25 @@
-const {expect} = require('chai')
 
-function print_color_map() {
+function getColorMap() {
     const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
     const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+    let colorMap=[];
     for (let i = 0; i < majorColors.length; i++) {
         for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
+            //console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
+            let currentObject={number:i*5+j,majorColor:majorColors[i],minorColor:minorColors[i]};
+            colorMap.push(currentObject);
         }
     }
-    return majorColors.length * minorColors.length;
+    return colorMap;
 }
 
-result = print_color_map();
-expect(result).equals(25);
-console.log('All is well (maybe!)');
+let colorMap = getColorMap();
+function printColorMap(){

```

Comment:
No test for the formatting (alignment of the `|` symbols on each row)? Try adding such a test

Reply:
I will add it